dm/syncer/compactor.go

```diff
@@ -165,6 +166,17 @@ func (c *compactor) compactJob(j *job) {
 	}
 
 	key := j.dml.identifyKey()
+
+	failpoint.Inject("DownstreamIdentifyKeyCheckInCompact", func(v failpoint.Value) {
```

Comment:
Does this Failpoint only use to output some debug log? If we want the test case fail, can we panic here if `if err != nil || value > upper {` condition satisfied.

Reply:
That's a good idea. It can omit checking the log.